pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/PraktiskInformasjonEtteroppgjoerUfoeretrygd.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.pensjon.brev.maler.vedlegg
+
+import no.nav.pensjon.brev.api.model.vedlegg.PraktiskInformasjonEtteroppgjoerDto
+import no.nav.pensjon.brev.maler.fraser.vedlegg.EttersendeDokumentasjon
+import no.nav.pensjon.brev.maler.fraser.vedlegg.InnledningPraktiskInformasjon
+import no.nav.pensjon.brev.maler.fraser.vedlegg.InntekterSomKanHoldesUtenforEtteroppgjoeret
+import no.nav.pensjon.brev.maler.fraser.vedlegg.SlikBetalerDuTilbake
+import no.nav.pensjon.brev.template.LangBokmalNynorskEnglish
+import no.nav.pensjon.brev.template.Language.*
+import no.nav.pensjon.brev.template.createAttachment
+import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
+import no.nav.pensjon.brev.template.dsl.newText
+
+// TODO: Conditional - this attachment is mandatory with letter UT_EO_FORHAANDSVARSEL_FEILUTBETALING_AUTO
+// The conditional for showing the letter is: ResultatEO or ResultatForrigeEO = 'tilbakekr'
+
+@TemplateModelHelpers
+val vedleggPraktiskInformasjonEtteroppgjoerUfoeretrygd =
+    createAttachment<LangBokmalNynorskEnglish, PraktiskInformasjonEtteroppgjoerDto>(
```

Comment:
For å ha et vedlegg uten dataklasse bytter du `PraktiskInformasjonEtteroppgjoerDto` med `Unit`.